draft-ppm-protocol.md

```diff
@@ -581,25 +585,26 @@ To encrypt an input share, the client first generates an HPKE
 {{!I-D.irtf-cfrg-hpke}} context for the aggregator by running
 
 ~~~
-enc, context = SetupBaseS(pk, "pda input share" || server_role)
+enc, context = SetupBaseS(pk, "pda input share" || task_id || server_role)
```

Comment:
Does it make sense to replace `server_role` with the index of the input share in the report struct? This may be useful for protocols with more than two aggregators.

Reply:
We punted this to a separate issue.